### PR TITLE
Issue #13213: Remove ok comments from descendanttoken package

### PR DESCRIPTION
**Pull request by Rohanraj123**
part of #13213 

Link to check documentation :  [clickHere](https://checkstyle.sourceforge.io/checks/misc/descendanttoken.html#DescendantToken)

Proof that all suppressions has removed : 
```
rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (master)
$ grep descendanttoken config/checkstyle-input-suppressions.xml
            files="checks[\\/]descendanttoken[\\/]InputDescendantTokenIllegalTokens.java"/>
            files="checks[\\/]descendanttoken[\\/]InputDescendantTokenIllegalTokens5.java"/>
            files="checks[\\/]descendanttoken[\\/]InputDescendantTokenIllegalTokens6.java"/>
            files="checks[\\/]descendanttoken[\\/]InputDescendantTokenReturnFromFinally2.java"/>
            files="checks[\\/]descendanttoken[\\/]InputDescendantTokenTestLimitedToken.java"/>

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (master)
$ git checkout issue-13213-descendanttoken
Switched to branch 'issue-13213-descendanttoken'

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (issue-13213-descendanttoken)
$ grep descendanttoken config/checkstyle-input-suppressions.xml

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (issue-13213-descendanttoken)
$ echo $?
1
```

### CODE REVIEW

**Review by romani — APPROVED** [review]
Ok to merge

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/descendanttoken/InputDescendantTokenTestLimitedToken.java [hunk 1]
```diff
@@ -7,8 +7,6 @@
 */
 
 package com.puppycrawl.tools.checkstyle.checks.descendanttoken;
-
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/descendanttoken/InputDescendantTokenTestLimitedToken.java:10]: Please restore empty line between package declaration and class declaration.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/descendanttoken/InputDescendantTokenTestLimitedToken.java (hunk 1 above)
Comment (Rohanraj123) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/descendanttoken/InputDescendantTokenTestLimitedToken.java:10]: did it sir !

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**